airflow/jobs/base_job.py

```diff
@@ -175,6 +176,17 @@ def kill(self, session=None):
     def on_kill(self):
         """Will be called when an external kill command is received."""
 
+    @retry_db_transaction
+    def handle_db_transaction_with_session(self, task_function, session):
+        return task_function(session)
+  
+    def handle_db_task(self, task_function):
+        try:
+            with create_session() as session:
+                return self.handle_db_transaction_with_session(task_function, session)
+        except OperationalError:
+            raise
```

Comment:
Why? This does nothing.

Reply:
You are right. I thought that I saw similar construct in code, however, I can not find it.
Nevertheless, I will remove this part.